benchmark/Cargo.toml

```
@@ -7,7 +7,6 @@ edition = "2021"
 
 [features]
 default = []
-unchecked = ["vibrato/unchecked"]
 
```

```suggestion
unchecked = []
```

Could you keep the `unchecked` feature in the benchmark tool since it is used to specify `Dictionary::read_unchecked()`?